src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -114,6 +117,12 @@ SettingsLoader::SettingsLoader(const std::string_view& userJSON, const std::stri
         }
     }
 
+    if (!_ignoredNamespaces.empty())
+    {
+        _removeIgnoredProfiles(inboxSettings);

```

Comment:
This part still isn't complete.

We need to keep thee profile stubs in JSON *even if the generator successfully completed*, but didn't generate the same profiles.

As a user, I should be able to:

1. Install Ubuntu
2. Customize Ubuntu in Terminal
3. Uninstall Ubuntu
4. (wait)
5. Reinstall ubuntu
6. **Retain my original customizations.**

The current implementation only works for the explicit "disabledProfileSources" case...